### ISSUE TITLE
install.sh hangs on macos with xcode 11.2

### ISSUE THREAD

**Issue by mellery451** [snare/voltron#261]
I've been using voltron with lldb/macos, but needed to reinstall b/c it stopped working. the install generates a warning and then is stuck and must be SIGTERMed:

```
Installing collected packages: voltron
  Found existing installation: voltron 0.1.7
    Uninstalling voltron-0.1.7:
      Successfully uninstalled voltron-0.1.7
    Running setup.py install for voltron ... done
Successfully installed voltron-0.1.7
+ GDB_INIT_FILE=/Users/ellery/.gdbinit
+ '[' -e /Users/ellery/.gdbinit ']'
+ sed -i.bak /voltron/d /Users/ellery/.gdbinit
+ '[' -z ']'
+ GDB_ENTRY_FILE=/Users/ellery/Library/Python/2.7/lib/python/site-packages/voltron/entry.py
+ echo 'source /Users/ellery/Library/Python/2.7/lib/python/site-packages/voltron/entry.py'
+ '[' 1 -eq 1 ']'
++ /usr/bin/lldb -Qxb --one-line 'script import platform; print(".".join(platform.python_version_tuple()[:2]))'
++ tail -1
warning: ignoring unknown option: -Qxb
error: module importing failed: inconsistent use of tabs and spaces in indentation (lldbinit.py, line 160)
  File "temp.py", line 1, in <module>

```

LLDB version:

```
$ lldb --version
lldb-1100.0.30.6
Apple Swift version 5.1.2 (swiftlang-1100.0.278 clang-1100.0.33.9)
```

**Comment by mellery451** [comment]
I think it's possible t`lldb` changed the way options are parsed (possibly: https://github.com/apple/swift-lldb/commit/65f9fc2eaab770a274776106425584e021aa7ae2). When I change all instances of `-Qxb` to `-Q -x -b` in the install script, I now fail with a different error:

```
+ LLDB_PYTHON=/Applications/Xcode.app/Contents/Developer/usr/bin/lldb
+ LLDB_PYTHON=/Applications/Xcode.app/Contents/Developer/usr/bin/lldb3.7
+ /Applications/Xcode.app/Contents/Developer/usr/bin/lldb3.7 -m pip install --user --upgrade six
./install.sh: line 171: /Applications/Xcode.app/Contents/Developer/usr/bin/lldb3.7: No such file or directory
```
..so it seems like maybe the version/executable detection is possibly broken with this version.

**Comment by StellarTime** [comment]
Also having this problem and tried to install it manually according to the instructions. 

For me, it seems that my Xcode 11 has its own Python3.7 for `lldb`, inside `/Applications/Xcode.app/....../bin/python3.7`,  with its modules in `../lib/python3.7/site-packages`. But running `/Application/.../bin/python3.7 -m pip install voltron` installed Voltron into `/Library/Python/3.7/site-packages`. And when inside `lldb`, Python does not look at this place for modules, i.e. `command script import /path/to/voltron/entry.py` is giving me `No module named 'blessed'`.


Xcode version

```
$ /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -version                                                                                                      
Xcode 11.0
Build version 11A420a
```

LLDB version

```
$ lldb --version                                                                                                                                                           
lldb-1100.0.28.19
Apple Swift version 5.1 (swiftlang-1100.0.270.13 clang-1100.0.33.7)
```

**Comment by zcutlip** [comment]
Thanks for reporting this issue. I'm having a look now.

The problem appears to be twofold:

- As described above `lldb` no longer likes `-Qxb` so we have to do `-Q -x -b`
- Also as described above lldb's `sys.executable` now reports `lldb` instead of python,

@StellarTime I tested installing as user which puts packages in ~/Library/Python, and lldb/voltron seemed happy.  I haven't looked at what happens when you install to /Library/Python, yet.

**Comment by zcutlip** [comment]
I merged PR #262, which hopefully addresses this issue. I'll go ahead and close it. If there are further troubles, please open a new issue.

**Comment by mellery451** [comment]
👍 - tested the latest and it works well for me.
